src/container/History/index.tsx

```diff
@@ -26,15 +54,18 @@ const History = () => {
         walletStore: { puzzleHash, isAblyConnected },
     } = rootStore
 
-    const groupedHistory = useMemo(
-        () =>
-            groupBy(history, (item) =>
-                formatDistanceToNowStrict(item.updatedAt, {
-                    addSuffix: true,
-                })
-            ),
-        [history]
-    )
+    const groupedHistory = useMemo(() => {
+        const option: IOptions = {
+            addSuffix: true,
+            locale: langItems.filter(
+                (item) => item.locale === i18n.language
+            )?.[0]?.file,
+        }
+
+        return groupBy(history, (item) =>
+            formatDistanceToNowStrict(item.updatedAt, option)
+        )
+    }, [history])
```

Comment:
sorry here add i18n.language, because you use `useTranslation` is in react lifecycle